packages/wonder-blocks-modal/src/util/find-focusable-nodes.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+/**
+ * List of elements that can be focused
+ * @see https://www.w3.org/TR/html5/editing.html#can-be-focused
+ */
+const FOCUSABLE_ELEMENTS =
+    'a[href], details, input, textarea, select, button:not([aria-label^="Close"])';
+
+export function findFocusableNodes(
```

Comment:
I see this is pretty similar to what we have in `wb-popover`. Probably we can abstract this more but it can be done separately. (No change required). In fact, I can see now that we have some shared utils with Modal, Popover and Tooltip that could be abstracted out into a separate (private) package.